village/src/components/Smurfs.js

```diff
@@ -1,22 +1,22 @@
-import React, { Component } from 'react';
-
+import React from 'react';
 import Smurf from './Smurf';
 
-class Smurfs extends Component {
-  // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
+function Smurfs (props) {
 
-  render() {
-    return (
-      <div className="Smurfs">
-        <h1>Smurf Village</h1>
-        <ul>
-          { this.state.smurfs.map((smurf) => {
-            return <Smurf name={smurf.name} age={smurf.age} height={smurf.height} key={smurf.id} />;
-          })}
-        </ul>
-      </div>
-    );
-  }
+  return (
+    <div className="Smurfs">
+      <h1>Smurf Village</h1>
+      <ul>
+        {props.smurfs.map((smurf) => {
+          return (
+            <Smurf key={smurf.id} name={smurf.name} age={smurf.age} height={smurf.height} uploadedSmurfs={props.uploadedSmurfs}/>
```

Comment:
let's consider using `div tags` here since this is basically the definition of the smurf class component, instead of passing data via props into itself?